### PR TITLE
Remove SSH functionality from Proxstar

### PR DESCRIPTION
**Pull request by WillNilges**
Now that VNC is over `wss://`, we don't need it.

### CODE REVIEW

**Comment by WillNilges** [comment]
In regards to the TODOs, I can use Redis to store the targets file data :0

File: proxstar/tasks.py [hunk 1]
```diff
@@ -150,7 +151,11 @@ def process_expiring_vms_task():
                             vm.name, vm.id
                         )
                     )
-                    # send_stop_ssh_tunnel(vm.id) # TODO (willnilges): Remove target from targets file
+                    vmid = vm['vmid'] 
+                    vnc_token_key = f'vnc_token|{vmid}'
+                    vnc_token = redis_conn.get(vnc_token_key).decode('utf-8')
+                    delete_vnc_target(token=vnc_token)
+                    redis_conn.delete(vnc_token_key)
```
Comment (WillNilges) [inline proxstar/tasks.py:158]: This is probably unnecessary, to be honest, and would cause more problems than it's worth.

**Comment by WillNilges** [comment]
```
diff --git a/proxstar/tasks.py b/proxstar/tasks.py
index a94ff70..0bf006f 100644
--- a/proxstar/tasks.py
+++ b/proxstar/tasks.py
@@ -151,7 +151,7 @@ def process_expiring_vms_task():
                             vm.name, vm.id
                         )
                     )
-                    vmid = vm['vmid'] 
+                    vmid = vm['vmid']
                     vnc_token_key = f'vnc_token|{vmid}'
                     vnc_token = redis_conn.get(vnc_token_key).decode('utf-8')
                     delete_vnc_target(token=vnc_token)

```

Thank you black, very cool.

**Comment by WillNilges** [comment]
bro smh

**Review by jabbate19 — CHANGES_REQUESTED** [review]
Debug Print, Thats ab it

**Review by jabbate19 — APPROVED** [review]
Chom

**Comment by WillNilges** [comment]
gonna rebase this a little because I want to be able to bisect this if the tasks break.

**Pull request merged**